velox/exec/Operator.cpp

```diff
@@ -97,6 +97,26 @@ void Operator::maybeSetReclaimer() {
       Operator::MemoryReclaimer::create(operatorCtx_->driverCtx(), this));
 }
 
+void Operator::maybeSetQueryTraceWriter() {
+  // TODO: Make this branch unlikely.
+  if (const auto& queryTraceConfig =
+          operatorCtx_->driverCtx()->queryTraceConfig()) {
+    if (queryTraceConfig->queryNodes.count(operatorType()) != 0) {
```

Comment:
Shall we support to include plan node id?

Reply:
Yes, using plan node ID is more fine-grained. I've refactor it to make it only supports plan node ids.